scripts/build

```diff
@@ -5,11 +5,14 @@ cd $(dirname $0)/..
 
 source ./scripts/version.sh
 mkdir -p build/images
+
 ./scripts/build-binary
-./scripts/build-windows-binary
 ./scripts/build-images
-./scripts/build-windows-images
 ./scripts/package-bundle
-./scripts/package-windows-bundle
-./scripts/dev-runtime-image
-./scripts/build-image-test
+if [ "${GOARCH}" != "s390x" ]; then
+  ./scripts/build-windows-binary
+  ./scripts/build-windows-images
+  ./scripts/package-windows-bundle
+  ./scripts/dev-runtime-image
+  ./scripts/build-image-test
+fi
```

Comment:
I am not confident that we can reorder these like this without breaking some of the build dependencies in ways that will not cause tests or the pipeline to fail, but may cause some artifacts to not be published. I'm also curious why we wouldn't want dev-runtime-image and build-image-test on s390x.

Can you leave the order here as-is, and add arch checks to the Windows scripts that just immediately `exit 0` on the wrong arch?

Reply:
done in https://github.com/rancher/rke2/pull/2171/commits/75921506838b7dd92924ea953168af0522501cc8